### ISSUE TITLE
Question about LaPred in Tables I and II

### ISSUE THREAD

**Issue by caiocj1** [valeoai/MFEval#4]
Hi, congrats on the awesome work!

Just a question, what would be the difference in the evaluation of LaPred in Tab I (the GT det&track input, 0.588 mAP_f line) and in Table II (the GT map, 0.760 mAP_f line)?

I had understood that both of these lines were simply trained on curated data and evaluated with GT map access, which would explain why the corresponding Agentformer line is the same in both tables. So I was wondering why LaPred isn't the same.

Thanks in advance!

**Comment by caiocj1** [comment]
Also, I would like to just confirm how the conventional models are trained with real perception inputs:
1. Is the regression loss for example applied only on agents that are matched to GT ones?
2. What is used as the target future trajectory, is it the predicted agents' predicted future tracks or the GT agents' actual future positions?

**Comment by yihongXU** [comment]
> Hi, congrats on the awesome work!
> 
> Just a question, what would be the difference in the evaluation of LaPred in Tab I (the GT det&track input, 0.588 mAP_f line) and in Table II (the GT map, 0.760 mAP_f line)?



> I had understood that both of these lines were simply trained on curated data and evaluated with GT map access, which would explain why the corresponding Agentformer line is the same in both tables. So I was wondering why LaPred isn't the same.
> 
> Thanks in advance!

The 0.760 mAP is from LaPred with 6 modes and the 0.588 one is with only 5 modes, for the reason of fair comparison with AgentFormer.

**Comment by yihongXU** [comment]
> Also, I would like to just confirm how the conventional models are trained with real perception inputs:
> 
> 1. Is the regression loss for example applied only on agents that are matched to GT ones?

Yes. By the way, we have tried different matching strategies in our recent work https://arxiv.org/abs/2406.08113. Please have a check.
> 2. What is used as the target future trajectory, is it the predicted agents' predicted future tracks or the GT agents' actual future positions?

It's the matched GT agents' actual future positions, thus there's an one-to-one matching process to run.

**Comment by caiocj1** [comment]
Thank you, that's very clear!
